sql-database/failover-groups/add-single-db-to-failover-group-az-ps.ps1

```diff
@@ -0,0 +1,130 @@
+# Set variables for your server and database
+$subscriptionId = '<SubscriptionID>'
+$resourceGroupName = "myResourceGroup-$(Get-Random)"
```

Comment:
__Recommended__: Resource group names are not required to be universally unique, so `$(Get-Random)` can be removed.

Reply:
@sptramer you and I have previously discussed with the SQL team uses a randomized variable for Resource Groups - we want to keep this - which you previously agreed to

Reply:
So go ahead and keep it. As the review comment says, you are not required to make the changes that are marked as __Recommended__ for the PR to be merged.